packages/myst-transforms/src/enumerate.ts

```diff
@@ -49,6 +57,20 @@ function getDefaultNumberedReferenceLabel(kind: TargetKind | string) {
   }
 }
 
+function getDefaultNamedReferenceLabel(kind: TargetKind | string, hasTitle: boolean) {
```

Comment:
This function should eventually be moved to be added/accessible from the directives. Currently this is hard-coded in the transforms.